<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fastinsert</class>
 <widget class="QDialog" name="fastinsert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fast insert...</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>81</width>
     <height>461</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Load my objects (*)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Previus object</string>
   </property>
   <property name="text">
    <string>&lt; Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Next object</string>
   </property>
   <property name="text">
    <string>Next &gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(*) The objects are loaded from Objects directory of the main path of Look3D installation</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>9</y>
     <width>521</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layoutGL"/>
  </widget>
  <widget class="QCheckBox" name="view_wireframe">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>wireframe</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="view_solid">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadpackage">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load package</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddObject">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>fastinsert</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>fastinsert</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
